### PR TITLE
Artificial intelligence conference PR

### PR DESCRIPTION
**Pull request by mprotic123**
This is pull request is created in order to review the project "Capstone 1" from Microverse. In this project, I was building an artificial intelligence conference website. Two pages are built for this milestone: Home and About page. Link to a Loom video: [Click here](https://www.loom.com/share/238cf370862e47228bcfbe80b369d8c2?sharedAppSource=personal_library)

**Project requirements**
- Personalize topic
- Design guidelines: colors, typography, layout
- Build two pages: Home & About 
- Build desktop and mobile version
- Interactions: links and mobile menu
- Dynamic page

### CODE REVIEW

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+name: Linters
```
Comment (SajjadAhmad14) [inline .github/workflows/linters.yml:1]: Please use the linter for HTML, CSS, and JavaScript from [this](https://github.com/microverseinc/linters-config/tree/master/html-css-js) linter config.

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+test.md
```
Comment (SajjadAhmad14) [inline .gitignore:1]: Please add all unnecessary files and folders like `.vscode` to `.gitignore`.

File: README.md [hunk 3]
```diff
@@ -8,7 +8,7 @@ This is a capstone project to showcase the skills acquired the last 4 weeks stud
 
 ## Live Demo
```
Comment (SajjadAhmad14) [inline README.md]: The live demo link in README is broken. Please add an active link here.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,218 @@
...
+      <section class="featured-speakers">
+        <h2 class="heading">Featured Speakers</h2>
+        <hr class="short-underline" />
+        <ul class="speakers"></ul>
+        <div class="button-container">
+          <button type="button" class="more">SEE MORE <i class="fa fa-chevron-down"></i></button>
+        </div>
+      </section>
+    </main>
+    <footer class="footer">
```
Comment (SajjadAhmad14) [inline index.html:175]: The footer on the mobile view should have a white background.
Yours:
footer
Design:
footer-o

File: index.html [hunk 5]
```diff
@@ -0,0 +1,218 @@
...
+            <p class="program-description">
+              This second edition aims at joining
+              both academy and industry by covering
+               not only basic and applied research,
+              but also philosophical and ethical
+               issues regarding the future of humanity in an AI world.
+            </p>
+          </li>
+        </ul>
+        <div class="whole-program"><a href="#">SEE THE WHOLE PROGRAM</a></div>
```
Comment (SajjadAhmad14) [inline index.html:161]: Please use the same color for all the sections of the home page from the design
see
.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,218 @@
...
+        <button type="button" class="join">Join Artificial Intelligence
+          International Conference</button>
+      </section>
+
+      <section class="featured-speakers">
```
Comment (SajjadAhmad14) [inline index.html:166]: Please follow the design for `Featured speakers` and `partner` sections on the Desktop view.
Your:
partner-y
Design:

partners
Your:
feature-y
Design:
feature-o

File: index.html [hunk 7]
```diff
@@ -0,0 +1,218 @@
...
+          <a href="#">Logout</a>
+        </li>
+        <li class="top-item">
+          <a href="#">MyPage</a>
+        </li>
+      </ul>
+    </header>
+    
+    <span class="hamburger"><i class="fas fa-bars"></i></span>
+    <nav class="navbar">
```
Comment (SajjadAhmad14) [inline index.html:44]: The navbar should have the same design for both `About` and `Home` pages on the desktop.
Your:
nav-y
Design:
nav-o

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Milan Protić, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
GitHub flow is followed. :heavy_check_mark:
README is professional. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html [hunk 8]
```diff
@@ -0,0 +1,218 @@
...
+               issues regarding the future of humanity in an AI world.
+            </p>
+          </li>
+        </ul>
+        <div class="whole-program"><a href="#">SEE THE WHOLE PROGRAM</a></div>
+        <button type="button" class="join">Join Artificial Intelligence
+          International Conference</button>
+      </section>
+
+      <section class="featured-speakers">
```
Comment (Mihndim2020) [inline index.html:166]: - [ ] As requested by the previous code reviewer which is also a requirement for this project, please make sure the feature speakers section is implemented as in the design, the design has two columns for the desktop version. Please kindly fix that.

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
## Hi @mprotic123, 

Great work on making the changes requested by a previous reviewer 👏 👏 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
stay-strong-youre-almost-there

 # Status: Requires Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 8 above)
Comment (mprotic123) [reply index.html:166]: Hi, @Mihndim2020 ! I am so sorry sir but I'm not sure that I understand what is wrong. Can you please explain it to me? 
Screenshot_1

Comment (SajjadAhmad14) [inline index.html:166]: `Featured Speakers` should be the same as in the design. So you can implement it using Grid. There are two columns and three rows in the design. You may have two rows and two columns. So kindly see [this](https://www.w3schools.com/css/css_grid.asp) resource and implement it.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Milan Protić, 
 ### Thank you for working on the changes suggested previously. :ok_hand:

## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by rukundoeric — APPROVED** [review]
Hi @mprotic123,

_Excellent job on this project. Keep it up 🎆_

_Congratulations! 🎉  on the splendid job 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_


Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

------
>As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**